src/main/java/net/minecraftforge/client/event/ScreenOpenEvent.java

```diff
@@ -19,9 +19,9 @@
 
 package net.minecraftforge.client.event;
 
+import net.minecraft.client.gui.screen.Screen;
 import net.minecraftforge.eventbus.api.Cancelable;
 import net.minecraftforge.eventbus.api.Event;
-import net.minecraft.client.gui.screen.Screen;
```

Comment:
Why?

Reply:
This is the result of letting IntelliJ organize imports. Will fix.